examples/particle-gibbs/script.jl

```diff
@@ -88,8 +86,8 @@ end
 
 # Here we use the particle gibbs kernel without adaptive resampling.
 model = NonLinearTimeSeries(θ₀)
-pgas = AdvancedPS.PG(Nₚ, 1.0)
-chains = sample(rng, model, pgas, Nₛ; progress=false);
```

Comment:
Minor comment, but why removing the `rng` argument ? That makes the doc reproducible (to some extent, rng implementations can change with julia versions)

Reply:
sorry, removed that when trying to get pgas to work. I edited the line now so that it is back to using rng as first argument, and also progress=false as in the original example.